nemo_text_processing/inverse_text_normalization/hi/data/time/second.tsv

```diff
@@ -0,0 +1,62 @@
+०१	एक
```

Comment:
are these different than the minute file?

Reply:
Yes, the minute.tsv and second.tsv files were the same, so I deleted one and renamed the other to minute_and_second.tsv. Now, I'm using just one file for both minutes and seconds.

Reply:
great -- thanks! like with hour, let's check to make sure we're implementing the same way as English

Reply:
just checked English and English also had a separate file for hour and minute.

Reply:
https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/inverse_text_normalization/en/taggers/time.py#L70